server/data_updates/00016_20181218-143214_archive.py

```diff
@@ -0,0 +1,126 @@
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-13T14:17:09.415371:50264206-eff2-4b13-bded-1eec7da3462b",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-17T12:04:17.596060:7ebd0c6a-ac0e-4289-adbb-9820aef8b4f7",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T18:17:25.149901:f2108169-91d6-42d1-b92d-a38c51337580",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T11:28:11.334794:8a67a371-d09a-4447-a795-eb3b1c7dac85",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T19:04:33.772440:ed22c69e-fa6f-447d-b23f-0b61c6930408",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-15T16:46:09.224970:c2366b2b-2664-4901-9258-9ad69be91699",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T16:31:56.698346:6209c7b3-32c0-4707-8821-8e8c8bc6107e",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T08:00:46.616456:969b9367-bf9e-4c7a-8c54-05245517ba9b",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T13:52:48.609682:dbdfab55-ec31-4230-b29a-8d9138bb5efe",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T11:39:48.012630:74a8f94e-c9c1-456a-a0d0-c5cb7c2d09f7",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T18:32:35.464312:07f77f7c-9a74-4ffe-b9e7-900fdefdfc77",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T13:49:21.517696:590dbacd-bea7-44cb-a3dd-9b28f38633e6",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-21T16:49:28.076869:2ab1bcc1-f2db-4d0c-a043-a01ca59bff55",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T19:04:34.016576:2c50815e-55a1-4a4d-94b1-81e7bc48e58a",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T18:17:25.404195:c23914d0-75f5-4f71-9ef5-b5fe0e3a95b1",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T13:52:48.895720:9a5326ee-4bad-4cba-85b5-187e6c99ea3c",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T16:47:49.111138:a92908dd-fa6b-493b-997c-cec54d5c427d",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-13T14:30:34.007235:c8627cc3-49e7-46ef-ac75-f74cecf694b8",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-14T09:58:04.733353:99d2511e-7753-4264-b48e-94fa209c4022",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T18:32:35.163559:562fe7a7-f9bc-485c-bd47-9f79eef69a83",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-23T13:52:48.546709:7a774ddc-64ea-4553-a56e-499229085066",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-17T11:57:50.849069:bcabee71-f1a7-4a89-b571-0e01604594ee",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T19:04:33.707202:2e47d154-f240-4040-9696-f98498d464bb",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-22T18:17:25.098125:0c3b5869-050d-449b-94a3-b750cddd269a",
+    "urn:newsml:fidelity-api.superdesk.pro:2017-11-16T13:35:33.689534:25f811bb-dcf3-4dd4-919e-17c94bc5e5af",
+]
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'archive'
+
+    def forwards(self, collection, mongodb_database):
+        archive_service = get_resource_service("archive_kill")
+        kill_service = get_resource_service("archive_kill")
```

Comment:
It is the same as `archive_service` I think it is typo.

Reply:
right, good catch, fixing it

Reply:
@ride90 it's fixed now